pulumi/infra/cache.py

```diff
@@ -50,6 +50,7 @@ def __init__(
                     cidr_blocks=["0.0.0.0/0"],
                 )
             ],
+            tags={"Name": f"{name}"},
```

Comment:
Make it easy to do resource lookups from the platform-infrastructure pulumi projects

Reply:
Unless you're augmenting the resource lookup search with other information, this tag alone won't provide a sufficiently narrow scope.

It would also be good for this PR comment to be a code comment on this line; once we have a more formalized relationship between stacks, such that we can simply refer to a stack output that gives us the security group ID directly, we won't need this tag anymore.

Reply:
Added the comment. And since we're only creating one redis security group, having it be a SG and tagged with Name= main-cache is sufficient for now.

Reply:
Oh, I just realized that you meant that if we have multiple deployments in the same AWS account. In that case you're right, we'd need to also look up the stack as well (which is a separate tag provided by auto-tagging)